handbook/product/roadmap.md

```diff
@@ -28,7 +28,14 @@ section Cloud
 %% section Code intel
 %%     TO DO :active, after release-3.21, 30d
 
-%% section Web
+section Web
+    Browser extension discoverability                         :done,    2020-09-28, 14d
+    Build new and improved extensions                         :active,   2020-10-12, 14d
+    Improve extensions development + docs                     :         2020-10-26, 14d
+    Code insights TBD                                         :         2020-11-09, 14d
+    Web nav updates                                           :         2020-11-09, 7d
+    Breadcrumbs                                               :         2020-11-16, 7d
+    Code insights TBD                                         :         2020-11-23, 14d
```

Comment:
I would leave this off for now since it's pretty open ended and the timeframe could be much longer. We can add it back and break it down in the coming weeks.